src/main/scala/com/twitter/finagle/postgres/values/JSONB.scala

```diff
@@ -0,0 +1,12 @@
+package com.twitter.finagle.postgres.values
+
+import java.nio.charset.Charset
+
+object JSONB {
+  def apply(string: String, charset: Charset = Charset.defaultCharset): JSONB = new JSONB(string.getBytes(charset))
```

Comment:
I think if we want to solve the charset problem, we need to solve it at the client level by storing the charset during the handshake. This solves it only for `JSONB`, and if we solve it for `String`s too, this API will change.